app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Users::ConfirmationsController < Devise::ConfirmationsController
+  # GET /resource/confirmation/new
+
+  # POST /resource/confirmation
+
+  # GET /resource/confirmation?confirmation_token=abcdef
+
+  # The path used after resending confirmation instructions.
+
+  # The path used after confirmation.
```

Comment:
- [ ] It will be nice to remove commented code from your code base as this makes your code base dirty and less readable.